index.html

```diff
@@ -10039,35 +10105,37 @@ <h4 id="Beast Shape - Medium Animal">Beast Shape - Medium Animal</h4>
             <tr>
                <td>Core
                </td>
-               <td>Pack Hunter
+               <td>Go for the Throat
                </td>
-               <td>Do additional Light damage for every ally adjacent to the enemy you are
-                  melee attacking
+               <td>Do a Medium+Dex melee attack against a creature.  
+                  If the creature is <a class="condition" data-tooltip-container>stunned</a>, <a class="condition" data-tooltip-container>prone</a>,
+                  <a class="condition" data-tooltip-container>immobilised</a>, <a class="condition" data-tooltip-container>unconscious</a>, or 
+                  <a class="condition" data-tooltip-container>paralysed</a>
+                  this deals Heavy+Dex damage.
                </td>
-               <td>Self
+               <td>E, Melee
                </td>
-               <td>Pass
+               <td>2
                </td>
             </tr>
             <tr>
-               <td>Core
+               <td>1
                </td>
-               <td>Go for the Throat
+               <td>Pack Hunter
                </td>
-               <td>Do a regular melee attack against a target that is either <a class="condition" data-tooltip-container>stunned</a>, <a class="condition" data-tooltip-container>prone</a>,
-                  or Imobolised. On a hit do Heavy+Dex damage
+               <td>Do an additional Light damage for every ally adjacent to the next enemy you damage with a melee attack this turn.
```

Comment:
On the next single target attack maybe? Idk if there are ways to get AoE melee in nature but ye, might be a bit much.

Reply:
Well, this would hurt the _first_ enemy in an AOE attack